.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+<!--
+Squash the commits, as appropriate. See .github/CONTRIBUTING.md for
+more information.
+
+Describe the relevant changes in the commit messages, if they need explaining.
+ 
+If the pull request introduces a new feature or an important bug fix, consider
+adding _one_ of the below tags, enclosed in square brackets, in its title:
+
+    FEATURE, FIX, DOC, WIP
+
+For some example:
+
+    [FEATURE] Parallel computation of cross-validation
+    [FIX] Don't crash NaiveBayes on input containing nan values
```

Comment:
Should we also advocate module/class tags like for commit messages?
E.g.
[FIX] naive_bayes: Don't crash on input containing nan values
or
[FIX] NaiveBayesLearner: Don't crash on input containing nan values

Which do we prefer? I like the module tags more, if the module is compact enough that the context of the change is clear. But that is not always the case... Allow the committer to decide?